ivan.kuznetsov/task-3/cmd/service/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"sort"
+
+	"github.com/kuzid-17/task-3/internal/config"
+	"github.com/kuzid-17/task-3/internal/jsonwriter"
+	"github.com/kuzid-17/task-3/internal/xmlparser"
+)
+
+func main() {
+	configFlag := flag.String("config", "", "Config file path")
```

Comment:
и  почему все игнорируют значение по умолчанию....(


```suggestion
	configFlag := flag.String("config", "config.yaml", "Config file path")
```